pkgs/development/interpreters/nickel/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nickel";
+  version = "0.1.0";
+
+  src  = fetchFromGitHub {
+    owner = "tweag";
+    repo = pname;
+    rev = "59bbb9036c0bacb03f0047f7d843b09cc0361b85"; # why is version failing?
```

The repository both has a branch and a tag name `0.1.0`. GitHub gets confused and returns an error.

This should work though:
```suggestion
    rev = "refs/tags/${version}";
```